odl/tomo/geometry/conebeam.py

```diff
@@ -993,248 +1003,210 @@ def __repr__(self):
     rotation_matrix = AxisOrientedGeometry.rotation_matrix
 
 
-class CircularConeFlatGeometry(HelicalConeFlatGeometry):
-
-    """Cone beam geometry with circular source curve and flat detector.
-
-    The source moves along a circle with radius ``src_radius`` in the
-    plane perpendicular to a fixed ``axis``. The detector reference
-    point is opposite to the source, i.e. in the same plane on a circle
-    with radius ``det_rad`` at maximum distance to the source. This
-    implies that it lies on the line through the source point and
-    the intersection of the ``axis`` with the azimuthal plane.
-
-    The motion parameter is the 1d rotation angle parameterizing source
-    and detector positions simultaneously.
-
-    In the standard configuration, the rotation axis is ``(0, 0, 1)``,
-    the initial source-to-detector vector is ``(0, 1, 0)``, and the
-    initial detector axes are ``[(1, 0, 0), (0, 0, 1)]``.
-
-    For details, check `the online docs
-    <https://odlgroup.github.io/odl/guide/geometry_guide.html>`_.
-
-    See Also
+def cone_beam_geometry(space, src_radius, det_radius, num_angles=None,
+                       short_scan=False, det_shape=None):
+    """Create a default fan or cone beam geometry from ``space``.
+
+    This function is intended for simple test cases where users do not
+    need the full flexibility of the geometries, but simply wants a
+    geometry that works.
+
+    The geometry returned by this function has equidistant angles
+    that lie (strictly) between 0 and either ``2 * pi`` (full scan)
+    or ``pi + fan_angle`` (short scan).
+    The detector is centered around 0 and has square pixels (3D case).
```

Comment:
I don't quite follow this. Why square pixels? Wouldn't the pixel shape depend on the geometry? Especially if `space` has anisotropic voxels